src/app/teams/teams.component.html

```diff
@@ -53,7 +53,7 @@ <h3>
               Request pending
             </button>
           </ng-container>
-          <button mat-raised-button color="primary" [routerLink]="['view', element.doc._id]" i18n>
+          <button mat-raised-button color="primary" [routerLink]="['view', element.doc._id]" *ngIf="element.userStatus!=='requesting'" i18n>
```

Comment:
**Manage Team** is seen user is already member. Other than that it should be **View Team** This button needs to be always available. Just the label changes.